src/core/ext/filters/client_channel/resolver/dns/c_ares/dns_resolver_ares.cc

```diff
@@ -405,7 +407,7 @@ void AresDnsResolver::MaybeStartResolvingLocked() {
 }
 
 void AresDnsResolver::StartResolvingLocked() {
```

Comment:
All of the messages in the other two files are prefixed with the address of the request object, but how will someone reading the log know which resolver the request is associated with?  I think we should have this method log a message indicating the address of the request it creates.

Reply:
Done. Good point. I originally tried to tie the resolver and the request together with the `addresses_` output param address. But logging the pending request is much simpler